activestorage/test/models/attachment_test.rb

```diff
@@ -104,7 +104,7 @@ class ActiveStorage::AttachmentTest < ActiveSupport::TestCase
     blob = create_blob
     @user.avatar.attach(blob)
 
-    signed_id_generated_old_way = ActiveStorage.verifier.generate(@user.avatar.id, purpose: :blob_id)
+    signed_id_generated_old_way = ActiveStorage.verifier.generate(@user.avatar.blob.id, purpose: :blob_id)
```

Comment:
@santib @georgeclaghorn this test was introduced by https://github.com/rails/rails/commit/31148cd6bef4f3a3059c51c587a8bff78a2e73e3#diff-3bd9077ea52e958612487234f70cabcab87e1014266ec902c1402b8452a736b4, a commit that shows both of you as the authors.

It isn't totally clear to me which part of the behavior is the "backwards compatibility" being covered. Does this change still keep this test covering that behavior?

Reply:
My reading of that is that the signed_id used to be generated with `ActiveStorage.verifier.generate` and now it is generated with `super`. I think this test is meant to verify that any old signed_ids generated with `ActiveStorage.verifier.generate` can still be used to find the blob.

You change looks good and still verifies that.